backend/entities/ParcelBuildings.cs

```diff
@@ -27,6 +27,11 @@ public class ParcelBuilding : BaseEntity
         /// get/set - The building located on the parcel.
         /// </summary>
         public Building Building { get; set; }
+
+        /// <summary>
+        /// get/set - Additional serialized metadata.
+        /// </summary>
+        public string LeasedLandMetadata { get; set; }
```

Comment:
The design called for this on the Parcel, but this didn't make sense to me since this metadata is only relevant to the building -> parcel relationship.

Reply:
Should be moved to the `Building`

Reply:
I'm not a big fan of that because then we have to track the array of notes separately from the actual parcel->building relationship.

Here is an example.

I have one building with 3 associated parcels, with one note for each. I then remove the first associated parcel. Now all of the note fields are out of sync with the actual relationships, and I have a note referencing a relationship that doesn't exist anymore.

Reply:
If you don't agree with the above let me know and we can do a call.